src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -54,18 +58,42 @@ class NodeProcessor
 
     /**
      * Regex to assert the a string is a valid PATH
+     *
+     * TODO: better regexp for this validation.
+     *
+     * We had /^((\/|..)?[-a-zA-Z0-9:_]+)+$/ which is too strict.
+     * We need to allow many more characters in the name, and we need to allow
+     * relative paths.
      */
-    const VALIDATE_PATH = '/^((\/|..)?[-a-zA-Z0-9:_]+)+$/';
+    const VALIDATE_PATH = '(((/|..)?[-a-zA-Z0-9:_]+)+)';
```

Comment:
this expression is rather pointless, requiring the path to contain once anywhere something of the range `[-a-zA-Z0-9:_]`. but the intention is quite wrong, and the expression above breaks many things. should we validate at all? how can we validate this in a sane way? it can be any path-valid character which is almost everything, and PathHelper::assertValidAbsolutePath does not work as the path may be relative in this case...

ideas what to do? drop the check? create PathHelper::assertValidRelativPath?